drivers/adc-dac/ad5592r/ad5592r-base.c

```diff
@@ -3,7 +3,7 @@
  *   @brief  Implementation of AD5592R Base Driver.
  *   @author Mircea Caprioru (mircea.caprioru@analog.com)
 ********************************************************************************
- * Copyright 2018(c) Analog Devices, Inc.
+ * Copyright 2018, 2020(c) Analog Devices, Inc.
```

Comment:
i would put these date changes in a separate commit

Reply:
I'll put these into a new commit

Reply:
I talked to the team and it seems we're very flexible about these dates in the header commentary. Consider it a matter of preferences and leave it as-is if you so wish.